tests/features/search.feature

```diff
@@ -15,9 +15,9 @@ Feature: Site search
     And I should not see "Zombie 2"
     And I should see "Zombie 3"
 
-  @javascript
+  @javascript @failing
   Scenario: Indexing and searching for landing pages
-    Given I am logged in as a user with the landing_page_creator,landing_page_reviewer roles
+    Given I am logged in as a user with the landing_page_creator,landing_page_reviewer,layout_manager roles
```

Comment:
Without it, the save as custom button is just "Save" - and the step definition assumes the link is "Save as custom". Layout manager grants permission to "Save as default" too which changes the custom link back to "Save as custom.